packages/fe-fpm-writer/templates/view/ext/CustomViewWithTable.xml

```diff
@@ -0,0 +1,29 @@
+<core:FragmentDefinition xmlns="sap.m" xmlns:core="sap.ui.core" xmlns:u="sap.ui.unified">
+    <Table id="<%- name %>">
+        <headerToolbar>
+            <OverflowToolbar>
+                <content>
+                    <Title text="Header Title" level="H2" />
+                    <ToolbarSpacer />
+                </content>
+            </OverflowToolbar>
+        </headerToolbar>
+        <infoToolbar>
+            <OverflowToolbar>
+                <Label text="Toolbar Label" />
+            </OverflowToolbar>
+        </infoToolbar>
+        <columns>
+            <Column>
+                <Text text="Column" />
+            </Column>
+        </columns>
+        <items>
+            <ColumnListItem>
+                <cells>
+                    <%- content %>
```

Comment:
That's too complex, I would say: no-one would know how to correctly supply `content`. 
It's easier to maintain it afterwards in the XML, thus I would neglect this parameter.

Reply:
Ok, thanks, will remove `content`.
If a user wants to generate a new eventHandler and a sample table, would it make sense to add a `Button` instead of `content` here to indicate/link that?

Reply:
Would it be possible to reuse this common functionality https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/common/defaults.ts#L41

Reply:
@tobiasqueck - thanks, yes would like to reuse it! 
Current approach: 
If `control` is custom pre-filled, a new fragment is generated. If set to `true`, then table fragment is created with `<%- content %>` filled by either eventHandler, or Text. 
Would that work, or is another solution needed?

Reply:
I always prefer keeping functionality consistent. For columns we either use the provided control, or generate sample code if `control === undefined`. We don't have the additional `true`. https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/column/index.ts#L46-L50

What would happen in your proposal if control is `undefined`? The fragment would be empty, correct? This is, of course, not wrong, however, I wonder if simple generated content is the better option because as the developer I can see in the preview that the generation worked.

Reply:
Is there an advantage in creating an example table as content compared to using the default fragment content i.e. the button?
I tend to say no because the developer creating a custom view and wants to create a table, has already something in mind, and won't need this example ( in addition, if we want to add a sample table, we should create an example with the table macro). Thus, I would recommend doing it the same as we do for columns, if a control is provided, use it as content, otherwise use the default fragment content. 
It would be consistent with our other generate functions and provides enough of an example to see in the preview that the generation was successful.

Reply:
@tobiasqueck 
I agree to the table as default content. The button is just an example in the FPM explorer, but not a real use case. On different view tabs of list report you usually show different lists. Charts in some cases.

Reply:
Updated generated xml fragment: https://github.com/SAP/open-ux-tools/pull/517/commits/f2e41eb0507915b08105ad35c044e3e4033d78b4#diff-372dd6c273bb1b939f13bfec4a5bc344527da25410d5fedab2f1f00e3fb8ee15R2